Clover/app/src/main/java/org/floens/chan/core/site/ThreadTime.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Clover - 4chan browser https://github.com/Floens/Clover/
+ * Copyright (C) 2014  Floens
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.floens.chan.core.site;
+
+public class ThreadTime {
+    public final int no;
+    public final long modified;
+
+    public ThreadTime(int no, long modified) {
+        this.no = no;
+        this.modified = modified;
```

Comment:
I'm not sure if `ThreadTime` is the best name for this, what I think you're tracking is a thread and the time it was last modified/updated. Could these attributes live on the `ChanThread` model?

Reply:
Yeah, the `ChanThread` model is only barely related to the pages model in that the OP number is used to relate them. I can refactor the ThreadTime moniker to something that's a bit less confusing, and put all the page related objects into one spot with a good label.